doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -1055,9 +1039,9 @@ Match over goals
    :n:`|- @ltac2_match_pattern`
       A pattern to match with the current goal
 
-   Note that unlike Ltac, only lowercase identifiers are valid as Ltac2
-   bindings, so that there will be a syntax error if one of the bound variables
-   starts with an uppercase character.
+   Note that unlike Ltac1, only lowercase identifiers are valid as Ltac2
+   bindings.  Ltac2 will report an error if a bound variables
+   that starts with an uppercase character.
```

Comment:
"if a bound variables" is incorrect grammar.  Maybe you meant
```suggestion
   bindings.  Ltac2 will report an error if you try to use a bound variable
   that starts with an uppercase character.
```
?

Reply:
A typo or omission on my part.